locale/en/docs/guides/nodejs-docker-webapp.md

```diff
@@ -95,11 +95,14 @@ WORKDIR /usr/src/app
 ```
 
 This image comes with Node.js and NPM already installed so the next thing we
-need to do is to install your app dependencies using the `npm` binary:
+need to do is to install your app dependencies using the `npm` binary. Please
+note that if you are using `npm` version 5 or later you will also want to copy
+`package-lock.json`:
 
 ```docker
 # Install app dependencies
 COPY package.json /usr/src/app/
+COPY package-lock.json /usr/src/app/
```

Comment:
I'd prefer `COPY package.json package-lock.json /usr/src/app/`. What do you think?